src/profile/UitpasCard/UitpasCard.tsx

```diff
@@ -16,8 +17,8 @@ type TProps = {
 const UitpasCard: FC<TProps> = ({ passHolder, isLarge }) => {
   const { width } = useWindowDimensions();
 
-  const activeUitpasCards = passHolder.cardSystemMemberships.filter(card => card.status === 'ACTIVE');
-  const [firstActiveCard] = activeUitpasCards;
+  const passHolderRegions = getPassHolderRegions(passHolder);
```

Comment:
Dit zou ook op 1 lijn kunnen, maar is misschien wel PP

Reply:
had gekund, maar `passHolderRegions` worden ook gemapt dus die hebben we wel degelijk nodig ;-)